### ISSUE TITLE
Loss function return shape

### ISSUE THREAD

**Issue by XPbIM3** [karolzak/keras-unet#48]
There is a jaccard_distance loss function implemented in this package and I'm suspicious about it.

Supposing You are feeding keras model with labels formed as (batch_size, dim_1, dim_2, class) the loss func should return either a (batch_size) array of  per sample losses or a scalar number - the loss over the whole batch.

This jaccard_distance :
`    intersection = K.sum(K.abs(y_true * y_pred), axis=-1)
    sum_ = K.sum(K.abs(y_true) + K.abs(y_pred), axis=-1)
    jac = (intersection + smooth) / (sum_ - intersection + smooth)`

returns an array of matrices if (batch_size, dim_1, dim_2, class) labeling is used.

**Comment by XPbIM3** [comment]
Sorry ,  Ive had a misleading concept in my head.  A return matrix of losses is totaly fine, thats not an issue.
